cmd/run.go

```diff
@@ -45,6 +46,10 @@ func newRunCmd(root *cobra.Command, conf app.Config, entrypoint func(ctx context
 			return errors.New("beacon-node-url does not contain a vaild URL")
 		}
 
+		if conf.ValidatorQueryChunkSize <= 0 {
+			return errors.New("validator query chunk size cannot be less or equal to 0")
```

Comment:
Minor nitpick, only because its externally facing text

```suggestion
			return errors.New("validator query chunk size cannot be less than or equal to 0")
```